src/gaen/nativeModule.ts

```diff
@@ -272,3 +272,7 @@ export const modifyLogEntry = (entry: SymptomLogEntry): Promise<void> => {
 export const getLogEntries = (): Promise<SymptomLogEntry[]> => {
   return symptomLogEntryModule.getSymptomLogEntries()
 }
+
+export const deleteLogEntry = (entry: SymptomLogEntry): Promise<void> => {
+  return symptomLogEntryModule.deleteSymptomLogEntry(entry.id)
```

Comment:
@aledustet It sends the ID to the native layer, but I can also use the ID in the JS side if that's easier for you

Reply:
No problem with that! I believe we added the method with the right signature so you might be even able to remove it altogether 😄